src/main/java/org/logstash/filters/NumericParserExecutor.java

```diff
@@ -0,0 +1,28 @@
+package org.logstash.filters;
+
+import org.joda.time.Instant;
+import org.logstash.Event;
+import org.logstash.filters.parser.TimestampParser;
+
+import java.io.IOException;
+
+class NumericParserExecutor implements ParserExecutor {
+  private TimestampParser parser;
+  public NumericParserExecutor(TimestampParser parser) {
+    this.parser = parser;
+  }
+
+  public Instant execute(Object input, Event event) throws IOException {
+    if (input instanceof String) {
```

Comment:
In the context of an Event's field value, we can be given Integer, Long, Double, a Map, etc. It's dangerous out here :)